### DISCUSSION TITLE
Can you pass the detail of which trigger was fired when you have multiple for a ScaledJob?

### DISCUSSION THREAD

**Discussion by jpspringall** [kedacore/keda#5206] [Q&A / Need Help] [answered]
Hi, 

Looking at the documentation I dont think this is possible, but I thought I would confirm

This is specific to the AWS SQS Trigger, but could theoretically apply to any trigger

I'm looking at the possibility of having multiple SQS triggers all triggering the same Scaled Job

That bit seems to work well, but I what I can't figure out is if it's possible to know which trigger was fired from within the container, by say passing an environment variable   

Cheers

James

**Comment by zroubalik** [comment, marked as answer]
It is not possible at the moment, because KEDA doesn't manipulate the workloads. It is an interesting usecase though, would you mind opening an issue for this?

**Comment by jpspringall** [comment]
Tis done @zroubalik #5218
